lib/utils/util.js

```diff
@@ -0,0 +1,59 @@
+/** @flow */
+
+import {spawn} from 'child_process';
+import {window, workspace} from 'vscode'
+
+const NODE_NOT_FOUND = '[Flow] Cannot find node in PATH. The simpliest way to resolve it is install node globally'
+const FLOW_NOT_FOUND = '[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g'
+
+function getPathToFlow(): string {
+	return workspace.getConfiguration('flow').get('pathToFlow')
+}
+
+export function isFlowEnabled() {
+	return workspace.getConfiguration('flow').get('enabled')
+}
+
+export function checkNode() {
```

Comment:
Yeah, I see that you've taken that approach and I think it's reasonable. My worry is that realistically most people won't install Flow themselves if they don't have to. Then, they will upgrade their editor plugin and suddenly their project has errors and they have to deal with that. Better to pay that cost up front when people are in setup mode, IMO, or just have the editor do it properly (again, discussed in that issue).